src/schema/v2/marketingCollections.ts

```diff
@@ -0,0 +1,180 @@
+    },
+    artistIds: {
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ artist_ids }) => artist_ids,
+    },
+    keyword: {
+      type: GraphQLString,
+      resolve: ({ keyword }) => keyword,
+    },
+    isFeaturedArtistContent: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      resolve: ({ is_featured_artist_content }) => is_featured_artist_content,
+    },
+    featuredArtistExclusionIds: {
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ featured_artist_exclusion_ids }) =>
+        featured_artist_exclusion_ids,
+    },
+    headerImageId: {
+      type: GraphQLString,
+      resolve: ({ header_image_id }) => header_image_id,
+    },
+    thumbnailId: {
+      type: GraphQLString,
+      resolve: ({ thumbnail_id }) => thumbnail_id,
+    },
+    showHeaderArtworksRail: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      resolve: ({ show_header_artworks_rail }) => show_header_artworks_rail,
+    },
+    showFeaturedArtists: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      resolve: ({ show_featured_artists }) => show_featured_artists,
+    },
+    artworkIds: {
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ artwork_ids }) => artwork_ids,
+    },
+  },
+})
+
+export const MarketingCollection: GraphQLFieldConfig<void, ResolverContext> = {
+  type: MarketingCollectionType,
+  description: "Marketing Collection",
+  args: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The slug or ID of the Marketing Collection",
+    },
+  },
+  resolve: (_root, { id, size }, { marketingCollectionLoader }) => {
+    if (!marketingCollectionLoader)
+      throw new Error(
+        "You need to pass a X-Access-Token header to perform this action"
+      )
+
+    return marketingCollectionLoader({ id, size })
+  },
+}
+
+export const MarketingCollectionsConnectionType = connectionWithCursorInfo({
+  nodeType: MarketingCollectionType,
+})
+
+export const MarketingCollectionsConnection: GraphQLFieldConfig<
+  void,
+  ResolverContext
+> = {
+  type: MarketingCollectionsConnectionType.connectionType,
+  description: "A list of MarketingCollections",
+  args: pageable({
+    slugs: {
+      type: new GraphQLList(GraphQLString),
+    },
+    artistId: {
+      type: GraphQLString,
+    },
+    category: {
+      type: GraphQLString,
+    },
+    sort: {
+      type: MarketingCollectionsSorts,
+    },
+    isFeaturedArtistContent: {
+      type: GraphQLBoolean,
+    },
+    size: {
+      type: GraphQLInt,
+    },
+  }),
+  resolve: async (_root, args, { marketingCollectionsLoader }) => {
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+
+    if (!marketingCollectionsLoader)
+      throw new Error(
+        "You need to pass a X-Access-Token header to perform this action"
+      )
+
+    const gravityArgs: {
+      page?: number
+      size: number
+      total_count: boolean
+      is_featured_artist_content?: boolean
+      artist_id?: string
```

Comment:
How does the `artistId` camel-cased argument passed-in get mapped to `artist_id` here?